KEYWORDS

```diff
@@ -88,6 +88,7 @@ Stride
 Syslog
 Techulus
 Telegram
+Threema
```

Comment:
I don't know what this file is used for but maybe (add/replace?) "Threema Gateway"?

```suggestion
Threema
Threema Gateway
```

Reply:
Done